languageFactories/language-factory-core/src/main/scala/cromwell/languages/util/ImportResolver.scala

```diff
@@ -122,7 +157,7 @@ object ImportResolver {
         else canonicalize(s"${relativeToValue.stripSuffix("/")}/$str").validNelCheck
       case None =>
         if (str.startsWith("http")) canonicalize(str).validNelCheck
-        else s"Cannot import '$str' relative to nothing".invalidNelCheck
+        else "Relative path".invalidNelCheck
```

Comment:
Is this a complete thought?

Reply:
It ends up looking ok (see the test errors). But I agree, it might be a candidate for improving at some point in the future